pycloudflare/models.py

```diff
@@ -124,7 +125,18 @@ def create_record(self, name, type, content, ttl=1, proxied=False,
         if type == 'MX':
             data['priority'] = priority
 
-        record = self.service.create_dns_record(self.id, data)
+        if type == 'SRV':
```

Comment:
Why not `elif`, because we don't need to check if `type` has already equalled `MX`?

P.S. I know you didn't do this, but we shouldn't be using `type` as a keyword.